composer.json

```diff
@@ -60,6 +60,7 @@
 		"saschaegerer/phpstan-typo3": "^1.9.1",
 		"seld/jsonlint": "^1.10.0",
 		"squizlabs/php_codesniffer": "^3.7.2",
+		"ssch/typo3-rector": "^1.3",
```

Comment:
For dev dependencies, please always let's use the full version here (and also the latest as far as possible), i.e. `^1.5.2`.

Reply:
Done.